IntoOOP/Bank/UI/Point.cs

```diff
@@ -0,0 +1,111 @@
+﻿namespace IntoOOP.Bank.UI;
+
+/// <summary>
+/// Точка на экране.
+/// </summary>
+public struct Point
```

Comment:
```suggestion
public readonly struct Point
```
Если все поля структуры неизменяемые, то и структуру надо сделать неизменяемой. Она будет работать быстрее.